### ISSUE TITLE
How to help page

### ISSUE THREAD

**Issue by Narigo** [jawu90/helpugee#1]
- How to participate: Add and edit data/content
- Links / Information about sponsoring / donations

- what are donations used for 
- what are we doing

**Comment by AnnieKfm** [comment]
You want to help?
These are the ways you can support us:
- Add new institutions and their services to your map
- Edit or delete outdated information within our map
- Help us to further develop Helpugee, please contact us here (Link zu Kontaktseite)
- Support to cover our running costs through donations here (Link zum Konto)

**Comment by Narigo** [comment]
Thanks! Need the "Link zum Konto"

**Comment by jawu90** [comment]
Good point - we will do a PayPal or betterplace fundraiser for the beginning and then see how to proceed on a longer-term plan - but will be having the link just when actually going online with something that we can ask donations for. Does that work for you?

**Comment by jawu90** [comment]
NEW VERSION: 

A community effort

This project was initiated to help people in need for leaving their home country. A tiny group of like-minded people pitched this project at the "Bayerwald Hackathon" and was lucky to motivate people with the necessary competencies for developing the first prototype. 

But in order to continuously optimize helpugee, gather new input and be up to date with all information, we need your help!

These are the ways in which you can support us:
- Add new institutions and their services to the map
- Edit or delete outdated information within the map
- Spread the word! Maybe your friends and family have more information to share here?
- Help us to further develop helpugee, please contact us here (Link zu Github)
- Donate to our fundraising campaign that helps cover running infrastructure costs. The link will be provided at a later stage.
